src/thunderbots/software/grsim_communication/grsim_backend.h

```diff
@@ -23,8 +23,7 @@ class GrSimBackend : public Backend
 
     ~GrSimBackend();
 
-    void sendPrimitives(
-        const std::vector<std::unique_ptr<Primitive>>& primitives) override;
+    void sendPrimitives(const std::vector<std::unique_ptr<Primitive>>& primitives);
```

Comment:
If this class no longer inherits from `Backend` (which I still think it should?) then we should give it a Javadoc comment.

Reply:
done